cfgmgr/intfmgr.cpp

```diff
@@ -1012,7 +1051,18 @@ bool IntfMgr::doIntfAddrTask(const vector<string>& keys,
             return false;
         }
 
-        setIntfIp(alias, "add", ip_prefix);
+        /*
+        Even though network device is created at this point by the SDK,
+        ipv6 setting on the iface might still be disabled
+        Thus don't proceed setting an Ipv6 addr unless this setting is set by SDK
+        */
+        if (!ip_prefix.isV4() && !g_ipv6Flag)
+        {
+            SWSS_LOG_ERROR("Tried to set ipv6 address %s to the iface %s, disable_ipv6 flag is set", ip_prefix.to_string().c_str(), alias.c_str());
+            return true;
+        }
```

Comment:
Why do we care about global IPv6 setting? We have a per interface check inside setIntfIp

Reply:
will remove